packages/client/tsconfig.json

```diff
@@ -2,5 +2,7 @@
   "extends": "@blockful/tsconfig/tsconfig.json",
   "compilerOptions": {
     "resolveJsonModule": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
```

Comment:
What is this about?

Reply:
it is required when importing the entities from Typeorm due to the decorator syntax